gobblin-hive-registration/src/main/java/gobblin/hive/metastore/HiveMetaStoreUtils.java

```diff
@@ -107,6 +109,15 @@ public static Table getTable(HiveTable hiveTable) {
     if (table.getTableType().equals(TableType.EXTERNAL_TABLE.toString())) {
       table.getParameters().put(EXTERNAL, Boolean.TRUE.toString().toUpperCase());
     }
+    if (hiveTable.getProps().contains(HiveConstants.RUNTIME_PROPS)) {
+      String runtimePropsString = hiveTable.getProps().getProp(HiveConstants.RUNTIME_PROPS);
+      for (String propValue : LIST_SPLITTER.splitToList(runtimePropsString)) {
+        List<String> tokens = LIST_SPLITTER.splitToList(propValue);
```

Comment:
I am probably not getting something here. If `runtimePropsString` was already split, it seems `propValue` would not contain any commas?